app/models/ticket.rb

```diff
@@ -72,8 +72,11 @@ def tickets_sold
   private
 
   def tickets_of_conference_have_same_currency
-    unless Ticket.where(conference_id: conference_id).all?{|t| t.price_currency == price_currency }
-      errors.add(:price_currency, 'is different from the existing tickets of this conference.')
+    tickets = Ticket.where(conference_id: conference_id)
+    unless tickets.count.zero? || (tickets.count == 1 && self == tickets.first)
```

Comment:
How about instead of this unless statement you use
`return true if condition`

Reply:
That's a nice idea, but I guess return if condition should be fine (for line 76). There is no need to return 'true'. What do you think?